README.md

```diff
@@ -67,6 +67,23 @@ OutlineView(data, children: \.children, selection: $selection) { item in
 
 ### Customization
 
+#### Children
+There are two types of `.children` parameters in the `OutlineView` initializers. You can either provide children for data items using a KeyPath pointing to an optional array of the same type, or a closure that provides an optional array based on the `ID` of the item.
+
+```swift
+// KeyPath option:
+OutlineView(data, children: \.children, selection: $selection) { item in
+  NSTextField(string: item.description)
+}
+
+// Closure option:
+OutlineView(data, children: { item in 
+  dataSource.childrenOfItem(item) 
+}) { item in
+  NSTextField(string: item.description)
+}
```

Comment:
```suggestion
// By passing a KeyPath to the children:
OutlineView(data, children: \.children, selection: $selection) { item in
  NSTextField(string: item.description)
}

// By providing a closure that returns the children:
OutlineView(data, children: { item in 
  dataSource.childrenOfItem(item) 
}) { item in
  NSTextField(string: item.description)
}
```

Reply:
The `selection` parameter is missing from the closure example.

Reply:
Also, https://github.com/Sameesunkaria/OutlineView/pull/10#discussion_r1083564808.

Reply:
Got it, and included the parameter order switch: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3